internal/core/src/query/visitors/ExecExprVisitor.cpp

```diff
@@ -2358,19 +2352,16 @@ ExecExprVisitor::ExecJsonContainsAllArray(JsonContainsExpr& expr_raw)
             for (auto&& e : val) {
                 json_array.emplace_back(e);
             }
-            for (auto index : tmp_elements_index) {
+            for (int index = 0; index < elements.size(); ++index) {
                 if (compareTwoJsonArray(json_array, elements[index])) {
```

Comment:
name follow same format, maybe CompareTwo.. is better.

Reply:
it's just an internal function.